### PR TITLE
Fix deployment scripts to use SCALA_VERSION

### PR DESCRIPTION
**Pull request by zeitos**
Leftover from the cross build PR.

### CODE REVIEW

File: job-server/test/spark.jobserver/JobSpecBase.scala [hunk 1]
```diff
@@ -28,7 +28,8 @@ trait JobSpecConfig {
       "spark.jobserver.max-jobs-per-context" -> MaxJobsPerContext,
       "akka.log-dead-letters" -> 0,
       "spark.master" -> "local[4]",
-      "context-factory" -> contextFactory
+      "context-factory" -> contextFactory,
+      "akka.test.single-expect-default" -> 6000
```
Comment (velvia) [inline job-server/test/spark.jobserver/JobSpecBase.scala]: Wait, this doesn't have to do with the script changes....

**Comment by velvia** [comment]
@zeitos thanks.  Might be time to refactor out some of the common script logic into a common.sh...

File: job-server/test/spark.jobserver/JobSpecBase.scala (hunk 1 above)
Comment (zeitos) [inline job-server/test/spark.jobserver/JobSpecBase.scala]: nope, you're right but I saw a couple of test failing on a slow machine that I have with 2.11

**Comment by zeitos** [comment]
I totally forgot about this, I'll try to catch up this week.

**Comment by velvia** [comment]
@zeitos thanks, LGTM, merging.

**Pull request merged**